hw/arm/aspeed: Switch fby35, grandcanyon to n25q00

peterdelevoryas opened the pull request:
This flash model is 128MB and includes the new BP and TB bits that Iris added. The additional testing verifies the vboot status output.

zhdaniel12 reviewed (dismissed):
nice, we have test case for machine!

chen-iris reviewed (dismissed):
Looks good! The modified line in m25p80 is probably too long but otherwise LGTM.

peterdelevoryas commented:
> Looks good! The modified line in m25p80 is probably too long but otherwise LGTM.

Ok I fixed that line, can you re-review? I changed the repo settings so that modifications to the PR require re-reviewing. Oops. I might disable that.
